Guardfile

```diff
@@ -65,6 +65,7 @@ guard :rspec, rspec_options do
     [
       rspec.spec.call("routing/#{m[1]}_routing"),
       rspec.spec.call("controllers/#{m[1]}_controller"),
```

Comment:
Probably, this line is not relevant anymore?

Reply:
Oh yeah, thanks!